### PR TITLE
Task#66691 Create order

### PR DESCRIPTION
**Pull request by Bui-Cong-Tri**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/66691)

## WHAT this PR does?
Thêm UI, Controller và validation cho create order từ cart

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by Bui-Cong-Tri** [comment]
Ready

File: app/Http/Requests/Order/OrderRequest.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'contact_id' => 'required|numeric',
+            'payment_method' => Rule::in(PaymentMethod::$types)
```
Comment (ghost) [inline app/Http/Requests/Order/OrderRequest.php]:
```suggestion
            'payment_method' => Rule::in(PaymentMethod::$types),
```

File: resources/js/app.js [hunk 2]
```diff
@@ -6,15 +6,16 @@ window.Alpine = Alpine
...
 
-import $ from "jquery";
-window.$ = window.jQuery = $;
+import $ from 'jquery'
+window.$ = window.jQuery = $
 
-$("[data-confirm]").on("click", function (e) {
-    var message = $(this).data("confirm");
+$('[data-confirm]').on('click', function (e) {
+  var message = $(this).data('confirm')
```
Comment (ghost) [inline resources/js/app.js]: indent 4 spaces

File: resources/lang/en.json [hunk 3]
```diff
@@ -4,5 +4,7 @@
   "Cancel": "Cancel",
   "Save changes": "Save changes",
   "Delete": "Delete",
-  "Edit": "Edit"
+  "Edit": "Edit",
+  "currency": "USD",
+  "Order": "Order"
```
Comment (ghost) [inline resources/lang/en.json]: indent 4 spaces

**Comment by Bui-Cong-Tri** [comment]
Ready

File: resources/css/app.css [hunk 4]
```diff
@@ -16,4 +16,11 @@
   .button.edit {
     @apply bg-blue-600  ring-blue-300 hover:hover:bg-blue-700 focus:ring-2 active:bg-blue-800;
   }
+  .button.disabled {
```
Comment (ghost) [inline resources/css/app.css]: indent 4 spaces

**Comment by Bui-Cong-Tri** [comment]
Ready

**Review by ghost — APPROVED** [review]
lgtm

**Pull request merged**